src/main/java/duke/handleInvalidArgs.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+
+/**
+ * This is a class to handle the invalid number of arguments for each of the class
+ */
+public class handleInvalidArgs {
```

Comment:
Perhaps can rename to InvalidArgsHandler?since we're suppose to use nouns for class names:)